charts/renku/requirements.lock

```diff
@@ -11,6 +11,9 @@ dependencies:
 - name: gitlab
   repository: https://swissdatasciencecenter.github.io/helm-charts/
   version: 0.3.0
+- name: renku-graph
+  repository: file://../../../renku-graph/helm-chart/renku-graph
```

Comment:
?

Reply:
it is, I forgot to put back the registry URL